packages/cli/src/commands/setup/graphql/features/fragments/__tests__/fragmentsHandler.test.ts

```diff
@@ -54,11 +55,11 @@ beforeAll(() => {
 
 afterAll(() => {
   process.env.RWJS_CWD = original_RWJS_CWD
-  jest.resetAllMocks()
-  jest.resetModules()
+  vi.resetAllMocks()
+  vi.resetModules()
 })
 
-test('all tasks are being called', async () => {
+test.skip('all tasks are being called', async () => {
```

Comment:
Guessing this is why this is still a Draft PR 🙂 
What's the problem with these tests?

Reply:
The problem is that we rely on mocking `@redwoodjs/project-config` but in this case this is not our direct dependency but an indirect dependency - as it is used within `@redwoodjs/cli-helpers`. Our project config package is transpiled to cjs so internally none of the require statements can be mocked out. This leads to the unmocked original code being executed instead of our mocked version.

See the first warning block within: https://vitest.dev/api/vi.html#vi-mock

We either have to have project config be ESM or we rewrite the mocks/tests. We're experimenting with the first option and will default to the later if we don't have success.

Reply:
Yeah, curious to see how the ESM work affects all of this :)